easybuild/easyconfigs/z/Zgoubi/Zgoubi-6.0.2-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'Zgoubi'
+version = '6.0.2'
+
+homepage = 'https://zgoubi.sourceforge.io'
+description = """Zgoubi is a ray-tracing code in use for charged particle beam dynamics simulations. 
+It can simulate beam dynamics in a large variety of machines and optical systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('GCC', '10.3.0'),
```

Comment:
Please use `binutils` as a build dependency instead:

```python
builddependencies = [
    ('binutils', '2.36.1'),
]
```

As it is now, EasyBuild wil try to find an easyconfig file named `GCC-10.3.0-GCCcore-10.3.0.eb` to resolve this dependency (which doesn't exist).